### DISCUSSION TITLE
How to configure CORS on Kestrel that runs Blazor WebAssembly

### DISCUSSION THREAD

**Discussion by FLAMESpl** [dotnet/aspnetcore#55381] [Q&A]
Hi,

how to confgure CORS on Kestrel that runs Blazor WebAssembly project? (The one that is started as part `dotnet run` command on client-side project).

All answers I have found on the internet treats about kestrel in ASP.NET backed Blazor (which I am not interested in).

Thanks in advance.

**Comment by gfoidl** [comment]
Do you see any error message in the browser console? I.e. what is blocked by CORS.

Can you give a more concrete sample of what you're trying to do?

The "hammer" is
```c#
app.UseCors(policy => policy
    .AllowAnyHeader()
    .AllowAnyMethod()
    .AllowCredentials()
    .AllowAnyOrigin());
```

Maybe https://github.com/dotnet/aspnetcore/issues/49871 is also relevant for you.

**Reply by FLAMESpl** [reply]
This is code for the server. In my project I do not have a server, just blazor WebAssembly client.
